pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -25,8 +25,8 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
     target: UnidentifiedWork,
     sources: Seq[TransformedBaseWork]): FieldMergeResult[FieldData] =
     FieldMergeResult(
-      fieldData = mergeItems(target, sources),
-      redirects = sources.filter { source =>
+      data = mergeItems(target, sources),
+      sources = sources.filter { source =>
```

Comment:
I think `getSourcesToMerge` or equivalent should be used consistently across all the rules - not sure if this is the only place where it's not yet used